lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/ibm-event-store.conf

```diff
@@ -0,0 +1,20 @@
+
+ibm.eventstore {
+
+  // If you are running IBM Project eventStore locally this will probably be a single IP and
+  // the port 5555. Use `ifconfig` to find your IP if you don't know it.
```

Comment:
I don't think `ifconfig` is a universal solution. IIRC `ipconfig` is the Windows equivalent.